pages/overview/allocation-strategy.md

```diff
@@ -0,0 +1,23 @@
+---
+description: What role does a Voting Strategy play in Allo?
+---
+
+# Allocation Strategy
+
+Every [ pool ](/overview/pool) in Allo is managed by an allocation strategy,
+which determines:
+
+1. Recipients - Who is eligible to receive funding from the pool?
+2. Allocation - How will funds be allocated? (grants committee, token voting, SBTs)
+3. Payouts - How will payouts be determined? (Quadratic funding, direct grant, request for proposal)
```

Comment:
Suggest rephrasing:

"Based on the allocation inputs, how will payouts be determined?"